docs/topics/circuit-breaker-mission-basic-interaction-spring-boot-tomcat.adoc

```diff
@@ -0,0 +1,96 @@
+= Interact with the Unmodified {SpringBoot} Booster
+
+Once you have the {SpringBoot} booster deployed, you have the following services running:
+
+//{app-name} = springboot-cb
+
+`{app-name}-greeting`::
+Exposes the `api/greeting` HTTP endpoint that the service client calls to get a personalized greeting response.
+
+This endpoint calls the remote `api/name` endpoint to obtain the current state of the `{app-name}-name` service.
+
+
+// maybe move this to the actual steps?
+If the remote endpoint is available, you receive the following greeting:
+----
+{"content":"Hello, World!"}
+----
+
+If service fails, you receive the predefined fallback response.
+----
+{"content":"Hello, Fallback!"}
+----
+
+`{app-name}-name`::
+Exposes the `api/name` HTTP endpoint that this application uses to simulate the failure of a service, and which returns
+
+* an HTTP Code `200` response when the service is available.
+* an HTTP Code `500` response when the service is unavailable.
+
+The following steps demonstrate how to verify the service availability, simulate a failure and receive a fallback response.
+
+NOTE: The following steps use the command line to interact with the service. Alternatively, you can use the web interface to perform the same steps.
+
+//TODO: add a warning not to use `http` as it may contain cached responses from the remote endpoint.
+. Use `curl` to execute a `GET` request against the `{app-name}-greeting` service. You can also use the `Invoke` button in the web interface to do this.
+// include image of the invoke button?
++
+[source,bash,options="nowrap",subs="attributes"]
+----
+$ curl http://{app-name}-greeting-{project-name}.1ab5.starter-us-east-1.openshiftapps.com/api/greeting
+{"content":"Hello, World!"}
+----
++
+// Add note about the Toggle button not working
+// no scaler implemented error if CLI used to scale down pod
+. To simulate the failure of the `{app-name}-name` service, use the `Toggle` button in the web interface. Alternatively, you can scale the pod running the `{app-name}-name` service down to 0 replicas using the web interface, or by executing the following command.
```

Comment:
Or you can do something like `curl -X PUT -H "Content-Type: application/json" -d '{"state": "fail"}' http://.../api/state`.

To recover, just change it to `{"state": "ok"}`.